t/e2e.t

```diff
@@ -347,7 +347,7 @@ sub spawn_server {
         exec @cmd;
         die "failed to exec $cmd[0]:$?";
     }
-    while (`netstat -na` !~ /^udp.*\s127\.0\.0\.1[\.:]$port\s/m) {
+    while (`ss --udp --all src "127.0.0.1:$port"` !~ /^\w+\s+\d+\s+\d+\s+127\.0\.0\.1:$port\s/m) {
```

Comment:
Instead of installing netstat(1) to our Ubuntu 16.04 image, this uses ss(1), which is installed by default.